sql/src/main/java/org/apache/druid/sql/calcite/planner/CalcitePlanner.java

```diff
@@ -502,4 +510,41 @@ void from(CalcitePlanner planner)
                                          + " to " + this);
     }
   }
+
+  /** Define some tool members and methods for hints. */
+  private static class HintTools
+  {
+    static final HintStrategyTable HINT_STRATEGY_TABLE = createHintStrategies();
+
+    /**
+     * Creates hint strategies.
+     *
+     * @return HintStrategyTable instance
+     */
+    private static HintStrategyTable createHintStrategies()
+    {
+      return HintStrategyTable.builder()
+                              .hintStrategy("broadcast", HintPredicates.JOIN)
```

Comment:
nit: is it really needed to burn in the `"broadcast"` literal here?

Reply:
note: I think this class should be moved in the `DruidHints` or something class you rename/restructure the `JoinHint` class to

Reply:
Changed to the enum name instead.